dash/include/dash/Coevent.h

```diff
@@ -0,0 +1,156 @@
+#ifndef COEVENT_H_INCLUDED
```

Comment:
looks like that should have been named 'DASH__COEVENT_H__INCLUDED'

Reply:
Yep, thanks! Will fix it.